src/client/services/cloak-service.js

```diff
@@ -132,6 +133,9 @@ function configureAndRun(roomId) {
             },
             resetFinished: () => {
                 dispatch(resetFinished());
+            },
+            initialGameParams: (params) => {
```

Comment:
To test this do we just need tests for the exported methods or also for the dispatches?